.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Nit: I don't think we should add non-project-specific files here. This belongs into ones global `.gitignore`.

Reply:
Can you elaborate @mxinden?  This `.gitignore` is at the project root.

Reply:
`DS_STORE` is a MacOS mechanism. It is not related to the libp2p blog. Thus it belongs in the `.gitignore` file in your home directory.

After a quick search, this post expands on the convention above: https://thomashunter.name/posts/2012-04-19-global-gitignore-vs-repository-gitignore